packages/analytics/analytics-utils/src/types/analytics-data.ts

```diff
@@ -0,0 +1,86 @@
+interface RecordEvent {
+  [field: string]: string | number;
+}
+
+interface Record {
```

Comment:
suggestion: you're overriding a generic here; should you rename this something more specific like `AnalyticsRecord` etc.?